dist/index.js

```diff
@@ -414,6 +434,23 @@ async function fetchPullRequest(context, repository, issue) {
   return pullRequest;
 }
 
+async function fetchApprovalReviewCount(context, pullRequest) {
+  const { octokit } = context;
+  const { number } = pullRequest;
+
+  logger.debug("Getting reviews for", number, "...");
+  let { data: reviews } = await octokit.pulls.listReviews({
+    owner: pullRequest.base.repo.owner.login,
+    repo: pullRequest.base.repo.name,
+    pull_number: number
+  });
+
+  reviews = reviews.filter(review => review.state === "APPROVED");
```

Comment:
When we wanted to see if everyone in the team is roughly doing the same amount of code reviews, someone figured out that GH allows you to give multiple approvals for the same PR, which quickly lead to our statistic being "slightly" skewed until we fixed the measurement script. So here we should probably only count the distinct authors, not the total number of approvals.

Reply:
That's a good catch, I made the wrong assumption.